nano/nano_wallet/entry.cpp

```diff
@@ -239,21 +289,45 @@ int main (int argc, char * const * argv)
 	{
 		QApplication application (argc, const_cast<char **> (argv));
 		boost::program_options::options_description description ("Command line options");
-		description.add_options () ("help", "Print out options");
-		description.add_options () ("config", boost::program_options::value<std::vector<std::string>> ()->multitoken (), "Pass configuration values. This takes precedence over any values in the node configuration file. This option can be repeated multiple times.");
+		// clang-format off
+		description.add_options () 
+			("help", "Print out options")
+			("config", boost::program_options::value<std::vector<std::string>> ()->multitoken (), "Pass configuration values. This takes precedence over any values in the node configuration file. This option can be repeated multiple times.")
+			("disable_backup", "Disable wallet automatic backups")
+			("disable_lazy_bootstrap", "Disables lazy bootstrap")
+			("disable_legacy_bootstrap", "Disables legacy bootstrap")
+			("disable_wallet_bootstrap", "Disables wallet lazy bootstrap")
+			("disable_bootstrap_listener", "Disables bootstrap processing for TCP listener (not including realtime network TCP connections)")
+			("disable_tcp_realtime", "Disables TCP realtime network")
+			("disable_udp", "Disables UDP realtime network")
+			("disable_unchecked_cleanup", "Disables periodic cleanup of old records from unchecked table")
+			("disable_unchecked_drop", "Disables drop of unchecked table at startup")
+			("fast_bootstrap", "Increase bootstrap speed for high end nodes with higher limits")
+			("batch_size",boost::program_options::value<std::size_t> (), "Increase sideband batch size, default 512")
+			("block_processor_batch_size",boost::program_options::value<std::size_t> (), "Increase block processor transaction batch write size, default 0 (limited by config block_processor_batch_max_time), 256k for fast_bootstrap")
+			("block_processor_full_size",boost::program_options::value<std::size_t> (), "Increase block processor allowed blocks queue size before dropping live network packets and holding bootstrap download, default 65536, 1 million for fast_bootstrap")
+			("block_processor_verification_size",boost::program_options::value<std::size_t> (), "Increase batch signature verification size in block processor, default 0 (limited by config signature_checker_threads), unlimited for fast_bootstrap");
```

Comment:
Looks like this duplicates the code in node's entry.cpp. Any way to share the options? Otherwise we'll have to update two places as we maintain options and descriptions.

Reply:
yeah, this should be better